src/Pages/ProfilePage/ProfilePage.js

```diff
@@ -0,0 +1,120 @@
+import { useLocation } from 'react-router-dom'
+import { generateUserInfo } from '../../utils/generateUserInfo'
+import { API } from '../../utils/api'
+import { randomImgAPI } from '../../utils/api'
+import { StyledProfileSection } from './styles/ProfileSection.styled.js'
+import { IcOutlineModeEdit } from '../../Icones'
+import { useDispatch, useSelector } from 'react-redux'
+import { Posts } from '../../Components'
+import axios from 'axios'
+import { useEffect } from 'react'
+import { getUsersPosts } from '../../Features/postsSlice'
+import { getUserInfo } from '../../Features/userSlice'
+import { likePost, unlikePost } from '../../Features/postsSlice'
+
+function ProfilePage() {
+	const location = useLocation()
+	const isLiked = useSelector(state => state?.posts?.isLiked)
+
+	const isUnliked = useSelector(state => state?.posts?.isUnliked)
+
+	// console.log(window.location.origin + location.pathname) //Future, use for url
+	const id = location.pathname.split('/')[2]
```

Comment:
suggestion: you can get this with `useParams`

Reply:
Okay, thanks for pointing this, will update in my next PR. Have already merged in local